src/test/java/testEatStreetDenis/pages/content/RestaurantsPage.java

```diff
@@ -0,0 +1,15 @@
+package testEatStreetDenis.pages.content;
+
+import ClasesToAllUs.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class RestaurantsPage extends BasePage {
+    public RestaurantsPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public WebElement restaurantsLabel() {
+        return findElementByXpath("//h1 [@class='restaurants-list_header ng-binding ng-scope']");
```

Comment:
Try to build your locators in more stable way
```suggestion
        return findElementByXpath("//h1 [Contains(@class, 'restaurants-list_header']");
```